public/static/docs/command-reference/import.md

```diff
@@ -15,17 +15,17 @@ import.
 usage: dvc import [-h] [-q | -v] [-o [OUT]] [--rev [REV]] url path
 
 positional arguments:
-  url         URL of Git repository with DVC project to download from.
+  url         URL of DVC or Git repository to download from.
```

Comment:
Is this from an actual change in the code? I don't see it in https://github.com/iterative/dvc/pull/3020/files. Please notice this new issue, BTW: iterative/dvc/issues/3077

Reply:
The change is that the Git repository no longer needs to have a '.dvc' project subdirectory.

Reply:
Right. I mean that `dvc import -h` doesn't say what you entered up here. (Just checked on the latest version, `0.80.0`.)

> Feel free to implement, BTW (iterative/dvc/issues/3077, stemming form your previous PR on that repo 🙂 )

Reply:
https://github.com/iterative/dvc/issues/3077 is now closed so I guess this thread can be resolved.

Reply:
Just noticed this. Yes, it seems version 0.82.0 (just out) does use this text. Thanks